### PR TITLE
Support TaxAreaLookups with multiple responses

### PR DESCRIPTION
**Pull request by rypit**
`TaxArea` requests sometimes return multiple `TaxAreaResults` 😳 -- which was slightly unexpected for us. This change makes the client pick the first/most relevant result in that case.

### CODE REVIEW

**Review by a-maas — APPROVED** [review]

File: lib/vertex_client/responses/tax_area.rb [hunk 1]
```diff
@@ -1,8 +1,16 @@
+require 'byebug'
```
Comment (drinks) [inline lib/vertex_client/responses/tax_area.rb]: note to delete this line

File: test/test_helper.rb [hunk 2]
```diff
@@ -1,4 +1,5 @@
 $LOAD_PATH.unshift File.expand_path("../../lib", __FILE__)
+require 'circuitbox'
```
Comment (drinks) [inline test/test_helper.rb]: just curious, why was this necessary?

**Review by drinks — APPROVED** [review]
LGTM! Thanks for doing this.

**Review by jfadgen — APPROVED** [review]

File: lib/vertex_client/responses/tax_area.rb (hunk 1 above)
Comment (rypit) [reply lib/vertex_client/responses/tax_area.rb]: Oh! Nice catch. Thanks!

File: test/test_helper.rb (hunk 2 above)
Comment (rypit) [reply test/test_helper.rb]: Because in the tax area test we only use the circuit if `circuitbox` is defined, which it is not when the test is run individually. The fact that the suite passes without this line currently is because of ordering. Will add a few more fixes as part of this PR too since the change is minor.
